.eslintrc.json

```diff
@@ -10,6 +10,7 @@
 	],
 	"rules": {
 		"eslint-plugin/consistent-output": "off",
+		"eslint-plugin/fixer-return": "off",
```

Comment:
Can we inline this instead?

Reply:
I don't think it's a useful rule. Often fixers will only run in a subset of cases, it isn't an error. We already assert that the fixer does something via output tests, and code coverage.

Reply:
If the rule asserted there was at least _one_ return statement, that might be useful.

Reply:
Filed as https://github.com/not-an-aardvark/eslint-plugin-eslint-plugin/issues/102